base/set.jl

```diff
@@ -191,6 +191,63 @@ function unique(f::Callable, C)
 end
 
 """
+    unique!(A::AbstractVector)
```

Comment:
this needs to be added to the stdlib doc index to show up in the rendered html docs